tests/unit/dataactvalidator/test_fsrs.py

```diff
@@ -0,0 +1,56 @@
+from unittest.mock import Mock
+
+from dataactvalidator import fsrs
+
+
+def newClient_call_args(monkeypatch, **config):
+    """Set up and create a newClient request with the provided config. Return
+    the arguments which were sent to the suds client"""
+    mock_client = Mock()
+    monkeypatch.setattr(fsrs, 'Client', mock_client)
+    config = {'fsrs_service': config}
+    monkeypatch.setattr(fsrs, 'CONFIG_BROKER', config)
+    fsrs.newClient()
+    return mock_client.call_args[1]
```

Comment:
Aaaah--yes, I see now. I was so intent on reading the letter of the docs that I completely missed what you were doing, so maybe your proposal would be clearer. Thanks!